cleansweep/views/admin.py

```diff
@@ -283,3 +283,18 @@ def _load_contacts(place, data):
         contacts += p.add_contacts(prows)
     db.session.commit()
     return contacts
+
+
+@app.route("/admin/settings/export", methods=['GET'])
```

Comment:
Please make the URL /admin/committee-structures/export and move this code to committees plugin.